webpack-subresource-integrity-embroider/index.js

```diff
@@ -8,7 +8,9 @@ class SubresourceIntegrityPlugin {
     compiler.hooks.done.tapPromise(
       "WriteSRIToIndexHtmlPlugin",
       async (stats) => {
-        const buildPath = stats.toJson().outputPath;
+        const statsJson = stats.toJson();
+        const buildPath = statsJson.outputPath;
+        const publicPath = statsJson.publicPath;
```

Comment:
Do you mind using destructuring for readability?

```suggestion
        const { buildPath, publicPath } = stats.toJson();
```

Reply:
will do